src/wat-lexer.h

```diff
@@ -55,11 +55,11 @@ struct IdTok {
   friend std::ostream& operator<<(std::ostream&, const IdTok&);
 };
 
-enum Signedness { Unsigned, Signed };
+enum Sign { NoSign, Pos, Neg };
```

Comment:
Is there any reason we need to have both `NoSign` and `Pos`? Can't we make the default `Pos`, and change it to `Neg` only when we take `-`?

Reply:
Hmm nevermind. We need to distinguish u32 vs. s32...